Production_scripts/Create_slf_lookups/lookup_gp_practice.R

```diff
@@ -84,12 +70,20 @@ pc <- read_spd_file() %>%
   # rename pc8
   rename(postcode = "pc8")
 
+
+# Data Cleaning ---------------------------------------
+
 data <-
-  data %>%
-  # join data and postcode data
-  left_join(pc, by = "postcode") %>%
+  opendata %>%
+  ## match cluster information onto the practice reference list ##
+  left_join(gpprac_file, by = c("gpprac", "postcode")) %>%
+  # sort by postcode
+  arrange(postcode) %>%
+  # match on geography info - postcode
+  left_join(pc_lookup, by = "postcode") %>%
   # rename hb2018
   rename(hbpraccode = "hb2018") %>%
+  # order variables
```

Comment:
I think `postcode` here refers to the postcode of the GP Practice? If so could we rename it to `gpprac_postcode` for clarity/readability, please?